packages/app-desktop/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "main.js",
   "private": true,
   "scripts": {
-    "dist": "yarn electronRebuild && npx electron-builder",
+    "dist": "yarn electronRebuild && npx electron-builder && ./rename.sh",
```

Comment:
I don't think that can work? electron-builder is going to publish and upload, so by the time you run your rename script it's too late.

Also we generally don't do shell scripts, but whatever works at this point

Reply:
Also this is supposed to run on Windows too and shell scripts don't work there (thus why we don't do shell scripts)